src/component/dashboard/dashboard.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import * as categoryActions from '../../action/category';
+import CategoryForm from '../category-form/category-form';
+import CategoryItem from '../category-item/category-item';
+
+class Dashboard extends React.Component {
+  render() {
+    const { categories, categoryCreate } = this.props;
+    return (
+      <div className = 'dashboard'>
+        <CategoryForm onComplete = { categoryCreate }/> {
+          categories.map((currentCategory, i) => 
+          <CategoryItem category = { currentCategory } key = { i }/>)
+        }
+      </div>
+    );
+  }
+}
+
+Dashboard.propTypes = {
+  categories: PropTypes.array,
+  categoryCreate: PropTypes.func,
+};
+
+const mapStateToProps = (state) => {
+  return {
+    categories: state,
```

Comment:
At Line 29, change `state` to `state.categories` and you'll get a step closer to solving your issue.